src/Microsoft.Azure.WebJobs.Host/Tables/TableAttributeBindingProvider.cs

```diff
@@ -365,5 +425,76 @@ public ResolvedTableAttribute(TableAttribute inner, IStorageTableClient client)
 
             internal IStorageTableClient Client { get; private set; }
         }
+
+        // IStorageTable --> IQueryable<T>
+        // ConverterManager's pattern matcher will figure out TElement. 
+        private class Table2IQueryableConverter<TElement> where TElement : ITableEntity, new()
```

Comment:
Why can't these converters be generic types like TableToQueryableConverter<IStorageTable, TElement> that derive from a converter interface that defines the contract? That would seem to make things clearer, and the types would be self describing. Also, that might make the searching for converters easier since you can look at interface types rather than searching by convention.